src/vuex/assets.module.js

```diff
@@ -42,6 +47,28 @@ export const actions = {
 
     commit(vuexTypes.SET_ASSETS, assets)
   },
+
+  async [vuexTypes.LOAD_STATS_QUOTE_ASSET] ({ commit, rootGetters, getters }) {
```

Comment:
LOAD_BUSINESS_STATS_QUOTE_ASSET, leave statsQuoteAsset as it was, use new

Reply:
renamed